impl/src/test/java/org/ehcache/internal/store/tiering/CacheStoreTest.java

```diff
@@ -80,8 +83,9 @@ public void testGetHitsAuthoritativeTier() throws Exception {
     when(cachingTier.getOrComputeIfAbsent(any(Number.class), any(Function.class))).then(new Answer<Store.ValueHolder<CharSequence>>() {
       @Override
       public Store.ValueHolder<CharSequence> answer(InvocationOnMock invocation) throws Throwable {
-        Number key = (Number) invocation.getArguments()[0];
```

Comment:
Looks like this updated changeset revived the unrelated formatting changes :smile: